### DISCUSSION TITLE
How to change publisher on existing extension

### DISCUSSION THREAD

**Discussion by msorens** [microsoft/vscode-discussions#508] [Extension Development QnA] [answered]
In the early days of my company we created one extension using an individual as the marketplace publisher. As I am now developing a second extension and properly setting up the company as a marketplace publisher, is it possible to migrate the first extension (with all its stats and history and whatever) into this new publisher?

**Comment by alefragnani** [comment, marked as answer]
Hi @msorens ,

If I remember correctly, you have to contact the VS Marketplace team directly (VSMarketplace@microsoft.com), in order to do that. Looking at vsmarketplace repo I found two posts that suggests this direction https://github.com/microsoft/vsmarketplace/issues/607 and https://github.com/microsoft/vsmarketplace/issues/133.

On the other hand, if _deprecation_ is an alternative for you, there is a live discussion about it https://github.com/microsoft/vscode-discussions/discussions/1, where you can ask for the deprecation in favor of the new extension.

Hope this helps

**Reply by msorens** [reply]
Thanks for the email address to the team; I have sent off a note so will see what they can do. Thank you!
